### PR TITLE
KAZOO-5347: ensure dst gets written properly

### PR DESCRIPTION
**Pull request by fenollp**

### CODE REVIEW

File: core/kazoo_config/src/kapps_config.erl [hunk 1]
```diff
@@ -853,7 +853,7 @@ migrate_config_setting(UpdatedFrom, Removed, To) ->
     case add_config_setting(To, Removed) of
         {'ok', UpdatedTo} ->
             {'ok', _} = kz_datamgr:save_doc(?KZ_CONFIG_DB, UpdatedTo),
-            {'ok', _} = kz_datamgr:save_doc(?KZ_CONFIG_DB, UpdatedFrom),
+            {'ok', _} = kz_datamgr:ensure_saved(?KZ_CONFIG_DB, UpdatedFrom),
```
Comment (lazedo) [inline core/kazoo_config/src/kapps_config.erl]: no, if someone updated before, we are overriding it.

Comment (fenollp) [inline core/kazoo_config/src/kapps_config.erl]: my testing says the opposite :7
it removed old field, kept the right one

Comment (lazedo) [inline core/kazoo_config/src/kapps_config.erl]: what tests did you make ?
client 1 -> get
client 2 -> get
client 1 -> set
client 2 -> set
value set from client 1 is lost

Comment (fenollp) [inline core/kazoo_config/src/kapps_config.erl]: what are you talking about? It's the config migration code.
Anyway, I'll implement the process I described on slack

Comment (lazedo) [inline core/kazoo_config/src/kapps_config.erl]: ah! you're right. i freak out every time i see `ensure_save`

**Pull request merged**